src/views/HardwareInstallation.vue

```diff
@@ -396,7 +501,32 @@ export default {
 			},
 		},
 	},
+	watch: {
```

Comment:
Die Verwendung eines Watchers deutet darauf hin, dass irgendwo die Reactivity gebrochen wurde oder ein anderes Problem vorliegt. Nach meiner Einschätzung ist für diesen Anwendungsfall kein Watcher notwendig.

Reply:
Ok.

Reply:
Das ist echt ein Problem, wofür ich keine andere Lösung gefunden habe. Wenn die store Daten nicht rechtzeitig da sind, dann werden die Auswahlfelder nicht gerendert:
TypeError: Cannot read properties of undefined (reading 'options'). Ich habe das damit gelöst, dass ich die erst rendere, wenn (this.hasData = true) ist. Für einen Gegenvorschlag bin ich dankbar.